widget/embedded/src/components/NotificationContent/NotificationContent.styles.ts

```diff
@@ -3,6 +3,7 @@ import { styled } from '@rango-dev/ui';
 export const Container = styled('div', {
   padding: '$10',
   width: '350px',
+  maxWidth: '90vw',
```

Comment:
Can you modify the popover to exclusively display within the boundaries of the widget?
![1](https://github.com/rango-exchange/rango-client/assets/120931880/106e1075-cfe8-4118-aff3-4947b506dca0)

Reply:
fixed